builder/amazon/ebs/acceptance/builder_acceptance.go

```diff
@@ -0,0 +1,32 @@
+package amazon_acceptance
```

Comment:
This file will get bundled into the Packer binary since it doesn't end with `_test.go`. Is that bad not really as no one is using this code outside of testing, but the bin may be bigger as there will be added dependencies. I would like to think about this :thinking:

Reply:
I couldn't make is as tests because of the structs. When it was a test and couldn't use the structs outside the package.

Reply:
Also, I couldn't put in the provisioned package because it causes a cyclic dependency 
```
# github.com/hashicorp/packer/helper/tests/provisioners
import cycle not allowed
package github.com/hashicorp/packer/helper/tests/provisioners (test)
	imports github.com/hashicorp/packer/builder/amazon/ebs/acceptance
	imports github.com/hashicorp/packer/command
	imports github.com/hashicorp/packer/provisioner/shell
	imports github.com/hashicorp/packer/command
FAIL	github.com/hashicorp/packer/helper/tests/provisioners [setup failed]
FAIL
```

Reply:
About the binary size, I figured out that Go doesn't include code not reachable from the main in the binaries. So Mock go files and these acceptance tests related files won't be included in the binaries.